test/TextToBinary.ModeSetting.cpp

```diff
@@ -190,15 +198,26 @@ INSTANTIATE_TEST_CASE_P(
         {CASE(OutputTriangleStrip), {}},
         {CASE(VecTypeHint), {96}},
         {CASE(ContractionOff), {}},
-    }),);
+    }), );
+
+INSTANTIATE_TEST_CASE_P(
+    TextToBinaryExecutionMode, OpExecutionModeTestV11,
+    ::testing::ValuesIn(std::vector<EnumCase<SpvExecutionMode>>{
+        {CASE(SubgroupSize), {12}},
+        {CASE(SubgroupsPerWorkgroup), {64}},
+        {CASE(LocalSize), {64, 1, 2}},
```

Comment:
Might be worth saying that in this test, LocalSize and below are spot checking features available in 1.0.
Above that line are features new to 1.1